vgsshow/src/main/java/com/verygoodsecurity/vgsshow/VGSShow.kt

```diff
@@ -334,31 +375,47 @@ class VGSShow constructor(
         )
     }
 
+    /**
+     * Used to create VGSShow instances with default and overridden settings.
+     *
+     * @constructor create VGSShow instance builder.
+     * @param context lifecycle owner context.
+     * @param id unique vault id.
+     */
     class Builder constructor(private val context: Context, private val id: String) {
 
         private var environment: VGSEnvironment = VGSEnvironment.Sandbox()
-
         private var host: String? = null
-
-        /** Specify Environment for the VGSCollect instance. */
-        fun setEnvironment(environment: VGSEnvironment): Builder = this.apply {
-            this.environment = environment
-        }
-
-        /** Sets the VGSCollect instance to use the custom hostname. */
-        fun setHostname(cname: String): Builder {
-            if (cname.isValidUrl()) {
-                host = cname.toHost()
-                if (host != cname) {
-                    logDebug("Hostname will be normalized to the $host")
-                }
+        private var port: Int? = null
+
+        /** Specify Environment for the VGSShow instance. */
+        fun setEnvironment(environment: VGSEnvironment) =
+            this.apply { this.environment = environment }
+
+        /**
+         * Sets the VGSShow instance to use the custom hostname. Custom host name can be localhost ip
+         * for ex. to use show with VGS-Satellite or custom hostname configured in VGS Dashboard.
```

Comment:
Sets the VGSShow instance to use the custom hostname. 
Also, the localhost IP can be used for VGS-Satellite for local testing.